File: nvxapp.client/src/nvx/ClientServer-Service/GestionePresenze/Az_Cfg/Models/az-cfg-model.ts
import { ModelResult } from "../../../ModelsBase/model-result";


export class Az_CfgModel {

  public id!: number;
  public idAz_Anagrafica!: number;
  public approvazioneTipo: TipoApprovazione;

}

export enum TipoApprovazione {
  SigleAdmin,
  AllAdmin,
  AllAdminHierarchy,
}


export class Az_Cfg_GetAll_InModel {
  
}
export class Az_Cfg_GetAll_OutModel extends ModelResult {

  public az_Cfg: Az_CfgModel;

}



export class Az_Cfg_Get_InModel {
  //public id: number;
}
export class Az_Cfg_Get_OutModel extends ModelResult {

  public az_Cfg: Az_CfgModel;

}
export class Az_Cfg_Put_InModel {

  public az_Cfg: Az_CfgModel;

}
export class Az_Cfg_Put_OutModel extends ModelResult {

  public az_Cfg: Az_CfgModel;

}

File: nvxapp.server.service/ClientServer-Service/GestionePresenze/Az_CfgService/Models/Az_CfgModel.cs
using nvxapp.server.data.Entities.Tenant;
using nvxapp.server.service.ClientServer_Service.ModelsBase;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace nvxapp.server.service.ClientServer_Service.GestionePresenze.Az_CfgService.Models
{
    public class Az_CfgModel
    {
        public int Id { get; set; }
        public int IdAz_Anagrafica { get; set; }
        public TipoApprovazione ApprovazioneTipo { get; set; }

    }

    public class Az_Cfg_GetAll_InModel
    {

    }
    public class Az_Cfg_GetAll_OutModel : ModelResult 
    {
        public Az_Cfg_GetAll_OutModel() 
        {
        
        }
    }


    public class Az_Cfg_Get_InModel
    {
        public  Az_CfgModel? Az_Cfg {  get; set; }
    }
    public class Az_Cfg_Get_OutModel : ModelResult
    {
        public Az_CfgModel? Az_Cfg { get; set; }
    }
    public class Az_Cfg_Put_InModel
    {
        public Az_CfgModel? Az_Cfg { get; set; }
    }
    public class Az_Cfg_Put_OutModel : ModelResult
    {
        public Az_CfgModel? Az_Cfg { get; set; }
    }

}
